pkg/model/awsmodel/network.go

```diff
@@ -253,13 +254,17 @@ func (b *NetworkModelBuilder) Build(c *fi.ModelBuilderContext) error {
 			CIDR:             fi.String(subnetSpec.CIDR),
 			Shared:           fi.Bool(sharedSubnet),
 			Tags:             tags,
+			DNS64:            fi.Bool(false),
 		}
 
 		if subnetSpec.IPv6CIDR != "" {
 			if !sharedVPC {
 				subnet.AmazonIPv6CIDR = b.LinkToAmazonVPCIPv6CIDR()
 			}
 			subnet.IPv6CIDR = fi.String(subnetSpec.IPv6CIDR)
+			// TODO: set this to true once NAT64 is in place
+			subnet.DNS64 = fi.Bool(false)
+			subnet.ResourceBasedNaming = fi.Bool(true)
```

Comment:
Because of CCM, we'll need to put an `IsKubernetesGTE("1.23")` condition on this.

Reply:
Is this a reasonable thing to happen when someone upgrades its cluster from k8s 1.22 to 1.23?

Reply:
This is only for ipv6 clusters, so there shouldn't be a problem here. We shouldn't support ipv6 without RBN,I think.
We can also use CCM 1.23 on k8s 1.22 clusters really. There is no other (significant) change to CCM at the moment.

Reply:
But, what we really need to do is set a minimum k8s version for IPv6.

Reply:
Agreed.